services/graph/pkg/service/v0/approleassignments.go

```diff
@@ -67,6 +68,10 @@ func (g Graph) CreateAppRoleAssignment(w http.ResponseWriter, r *http.Request) {
 		RoleId:      appRoleAssignment.AppRoleId,
 	})
 	if err != nil {
+		if merr, ok := merrors.As(err); ok && merr.Code == http.StatusForbidden {
+			errorcode.GeneralException.Render(w, r, http.StatusForbidden, err.Error())
```

Comment:
This should be `errorcode.NotAllowed.Render()`

Reply:
Done. Thanks.